dmoj/urls.py

```diff
@@ -235,9 +235,9 @@ def paged_list_view(view, name):
         url(r'^user/list$', api.api_v1_user_list),
         url(r'^user/info/(\w+)$', api.api_v1_user_info),
         url(r'^user/submissions/(\w+)$', api.api_v1_user_submissions),
-        url(r'^v2/', include([
-            url(r'user-info$', api.api_v2_user_info),
-        ])),
+#        url(r'^v2/', include([
+#            url(r'user-info$', api.api_v2_user_info),
+#        ])),
```

Comment:
Too many changes in one pull request, and since we have version control, we shouldn't leave code commented out.